server/tracedb/connection/connection.go

```diff
@@ -39,23 +39,25 @@ func (r ConnectionTestStepResult) IsSet() bool {
 }
 
 func IsReachable(endpoint string) (bool, error) {
-	address, err := url.Parse(endpoint)
-	if err != nil {
-		return false, err
-	}
+	if strings.HasPrefix(endpoint, "http://") || strings.HasPrefix(endpoint, "https://") {
```

Comment:
By checking if it's HTTP or not. It prevents problems when an IP is used as an endpoint.